materialized_sql_view/model/abstract_materialized_sql_view.py

```diff
@@ -0,0 +1,344 @@
+# -*- coding: utf-8 -*-
+
+import psycopg2
+import logging
+from openerp.osv import osv
+from openerp import SUPERUSER_ID
+from abc import ABCMeta, abstractmethod
+
+logger = logging.getLogger(__name__)
+ABSTRACT_MODEL_NAME = 'abstract.materialized.sql.view'
+
+
+class AbstractMaterializedSqlView(osv.AbstractModel):
+
+    """This class is an abstract model to help developer to
+       create/refresh/update materialized view.
+    """
+    _name = ABSTRACT_MODEL_NAME
+    _description = u"This is an helper class to manage materialized SQL view"
```

Comment:
Description is the verbose name of the record. i.e. res.partner -> Partner as in "A new Partner was created"

This is too long and doesn't make sense in that context. The docstring is enough of a description.